Node_Editor_Framework/Runtime/Framework/Core/ConnectionPort.cs

```diff
@@ -248,6 +248,8 @@ public void RemoveConnection (ConnectionPort port, bool silent = false)
 					() => NodeEditorUndoActions.DeleteConnection(port1, port2),
 					() => NodeEditorUndoActions.CreateConnection(port1, port2),
 					"Delete Connection");
+					
+				port.body.OnRemoveConnection(this, port);
 			}
```

Comment:
Can you move this down to the other callback? Only functional difference is the editor check, but is also more consistent.

Reply:
Yeah you're right! Just finished adjusting it, let me know if there's any other changes you'd prefer.